### PR TITLE
Elections improvements

### PR DESCRIPTION
**Pull request by g4s8**
#1564 
 - calculating election winner in java instead of xsl
 - removed `to-winner.xsl`

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1575](https://github.com/zerocracy/farm/issues/1575) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1575](https://github.com/zerocracy/farm/issues/1575) is assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2211732460); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/zerocracy/pm/staff/ElectionResult.java [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+     */
+    public String reason() throws IOException {
+        final int max = ElectionResult.max(
+            this.elections.get(0).nodes("//election").get(0)
+        );
+        return String.join(
+            "\n",
+            new Mapped<>(
+                entry -> String.format(
+                    "@%s (%f of %d):\n%s",
```
Comment (krzyk) [inline src/main/java/com/zerocracy/pm/staff/ElectionResult.java]: @g4s8 I think we should limit the decimals to just 2 places (`%.2f`) - current implementation does (at least that's what I see in the results)

File: src/main/java/com/zerocracy/pm/staff/ElectionResult.java [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+         * @return Score of this vote
+         */
+        public double score() {
+            return (double) this.weight * this.points;
+        }
+
+        @Override
+        public String toString() {
+            return String.format(
+                "+%f=%fx%d %s",
```
Comment (krzyk) [inline src/main/java/com/zerocracy/pm/staff/ElectionResult.java]: @g4s8 should be `%f` -> `%.2f`

File: src/main/java/com/zerocracy/pm/staff/ElectionResult.java [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+                        new Mapped<>(
+                            vote -> vote.score(),
+                            entry.getValue()
+                        )
+                    ).doubleValue(),
+                    max,
+                    String.join(
+                        "\n",
+                        new Mapped<>(
+                            vote -> String.format("  %s", vote.toString()),
```
Comment (krzyk) [inline src/main/java/com/zerocracy/pm/staff/ElectionResult.java]: @g4s8 in `String.format` using `.toString` is not needed

File: src/main/java/com/zerocracy/pm/staff/ElectionResult.java [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+            this.details = details;
+        }
+
+        /**
+         * Vote's score.
+         *
+         * @return Score of this vote
+         */
+        public double score() {
+            return (double) this.weight * this.points;
```
Comment (krzyk) [inline src/main/java/com/zerocracy/pm/staff/ElectionResult.java]: @g4s8 `(double)` is not needed, int will get upgraded to double here

**Comment by g4s8** [comment]
@krzyk should fixed now, please take a look

**Comment by krzyk** [comment]
@g4s8 thanks

**Comment by krzyk** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by g4s8** [comment]
@0crat status

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/farm/issues/1575#issuecomment-412352363))

@g4s8 This is what I know about this job in [C3NDPUA8L](https://www.0crat.com/p/C3NDPUA8L), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#1575](https://github.com/zerocracy/farm/issues/1575) is [in scope](http://www.zerocracy.com/policy.html#14) for 3days
  * The role is `REV`
  * The job is not assigned to anyone
  * The [budget](http://www.zerocracy.com/policy.html#4) is not set yet
  * These users are banned and won't be assigned:
    * @g4s8[/z](https://www.0crat.com/u/g4s8): This user reported the ticket
  * The job is waiting quality review verdict by @ypshenychka[/z](https://www.0crat.com/u/ypshenychka) for 9min
  * @krzyk[/z](https://www.0crat.com/u/krzyk) will receive 15 minutes after quality review
  * Job [footprint](https://www.0crat.com/footprint/C3NDPUA8L?q=%7Bjob%3A%27gh%3Azerocracy%2Ffarm%231575%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by 0crat** [comment]
The job [#1575](https://github.com/zerocracy/farm/issues/1575) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +20 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**